scan-manager/docs/setup_artifacts/upload_artifact.sh

```diff
@@ -2,17 +2,20 @@
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
-AWS_BUCKET=$1
+END_POINT=$1
+ACCESS_KEY=$2
+SECRET_KEY=$3
+BUCKET_NAME=$4
 
-if [[ -z $AWS_BUCKET  ]]; then
-    echo "ERROR: Missing s3://AWS_BUCKET "
+if [[ -z $BUCKET_NAME  ]]; then
+    echo "ERROR: Missing bucket"
     exit 1
 fi
 
-echo "make sure aws credentials are setup for upload to work"
 echo "uploading deployment files .."
 
-aws s3 cp --recursive $DIR/deployment $AWS_BUCKET/data/deployment
+mc alias set profile $END_POINT $ACCESS_KEY $SECRET_KEY
```

Comment:
Is there a pre-requisite list where we ask them to have `mc` pre-installed? If not, we should have it.

Reply:
agree here, minio should be optional

Reply:
Minio can't be optional. It supports S3 and Minio buckets. It should be the default.

Reply:
ok then Prajna says